examples/v1beta1/custom-family.yaml

```diff
@@ -0,0 +1,42 @@
+# This example provisioner will provision instances using an AMI that belongs to a custom AMIFamily
+# Keep in mind, that you're in charge of bootstrapping your worker nodes.
+
```

Comment:
(nit) When a YAML file has a `---` line, I put one at the start as well.

That helps people learn that `---` is a document start marker, not a separator.

Reply:
```suggestion
# This example NodePool provisions instances using an AMI that belongs to a custom AMIFamily
# Keep in mind that you're in charge of bootstrapping your worker nodes.
```